core/src/main/scala/akka/persistence/cassandra/query/EventsByTagFetcher.scala

```diff
@@ -113,6 +113,9 @@ private[query] class EventsByTagFetcher(
           else
             highestOffset = offs
 
+          // FIXME reproducer of issue #206
+          // Thread.sleep(30)
```

Comment:
Leave comment for documentation purposes, but it's not really a `FIXME` so remove that?